applications/GeoMechanicsApplication/tests/fixed_spatial_variation/README.md

```diff
@@ -0,0 +1,29 @@
+# Prescribed spatially varying fixed water pressure
+
+This test is a water pressure tests of a single UPwSmallStrainElement2D8N element.
+It checks whether fixed water pressure conditions stay fixed when using a spatially varying fixed water pressure. In this case, it means we use the `ApplyScalarConstraintTableProcess` with the following tables (x, water_pressure pairs) for the top and bottom sides of the element respectively:
+```
+Begin Table 1 X WATER_PRESSURE
+0.0000000000 0.0000000000
+1.0000000000 -5000
+End Table
+
+Begin Table 2 X WATER_PRESSURE
+0.0000000000 -40000
+1.0000000000 -30000
+End Table
+```
+
+## Setup
+- Constraints:
+    - Displacements on the bottom are fixed in all directions.
+    - Displacements on the sides are fixed in the X direction.
+    - Water pressures on the top are varying from 0 to -5000, and on the bottom from -40000 to -30000. These are fixed in time.
```

Comment:
"constant in time" would relieve the possible confusion with fixed in space.

Reply:
Agreed!